frontend/src/features/permits/pages/Payment/PaymentRedirect.tsx

```diff
@@ -70,7 +70,7 @@ const mapTransactionDetails = (
     approved: Number(motiResponse.trnApproved),
     authCode: motiResponse.authCode,
     cardType: motiResponse.cardType,
-    transactionDate: "TODO", // TODO
+    transactionDate: motiResponse.trnDate, // TODO
```

Comment:
I have left this date unformatted. I am passing what was given by moti pay. Seems like its in Pacific time?